cli_tools/gce_ovf_import/main.go

```diff
@@ -93,7 +93,18 @@ func buildImportParams() *ovfimportparams.OVFImportParams {
 	}
 }
 
-func runImport() (*daisy.Workflow, error) {
+// InstanceExportTool is the tool for instance import
+type InstanceExportTool struct {
+	service.CliToolWithLogging
+}
+
+// ActionType implements CliToolWithLogging
+func (t *InstanceExportTool) ActionType() service.ActionType {
+	return service.InstanceImportAction
+}
+
+// MainFunc implements CliToolWithLogging
+func (t *InstanceExportTool) MainFunc() (*daisy.Workflow, map[string]string, error) {
```

Comment:
Why not just Main()? Func is redundant as it's obvious it's a func. It shouldn't conflict with main().

Even better, Run() or similar would prevent any confusion